sdk/identity/identity/src/credentials/managedIdentityCredential.ts

```diff
@@ -375,7 +392,7 @@ export class ManagedIdentityCredential implements TokenCredential {
 
     try {
       // isEndpointAvailable can be true, false, or null,
-      // the latter indicating that we don't yet know whether
+      // If it's null, it means we don't yet know whether
```

Comment:
`the latter` has some extra cognitive load for non-English speakers. At least for me. I prefer to not use it if possible.